content/darkroom/masking-and-blending/blend-modes.md

```diff
@@ -5,44 +5,159 @@ weight: 20
+
+vividlight
+: This mode is an extreme version of overlay/softlight. Values darker than mid-grey are darkened; Values brighter than mid-grey are brightened. You will probably need to tone down its effect by reducing the opacity
+
+linearlight
+: This mode is similar to the effect of "vividlight".
+
+pinlight
+: This mode performs a darken and lighten blending simultaneously, removing mid-tones. It can result in artifacts such as patches and blotches.
+
+# color channel modes
+
+## Lab channels
+
+The following are available for blending in the Lab color space only
 
 Lab lightness
-: Only available with modules that work in the Lab color space, this blend mode mixes lightness from the input and output images, while color is taken unaltered from the input image. In contrast to “lightness” this blend mode does not involve any color space conversion and does not clamp any data. In some cases this blend mode is less prone to artifacts than “lightness”.
+: Mix the lightness from the input and output images, while taking the color channels (a and b) unaltered from the input image. In contrast to “lightness” this blend mode does not involve any color space conversion and does not clamp any data. In some cases this blend mode is less prone to artifacts than “lightness”.
+
+Lab a-channel
+: Mix the Lab "a" color channel from the input and output images, while taking the other channels unaltered from the input image.
+
+Lab b-channel
+: Mix the Lab "b" color channel from the input and output images, while taking the other channels unaltered from the input image.
 
 Lab color
-: Only available with modules that work in the Lab color space, this blend mode mixes Lab color channels a and b from the input and output images, while lightness is taken unaltered from the input image. In contrast to “color” this blend mode does not involve any color space conversion and does not clamp any data. In some cases this blend mode is less prone to artifacts than “color”.
+: Mix the Lab color channels (a and b) from the input and output images, while taking the lightness unaltered from the input image. In contrast to “color” this blend mode does not involve any color space conversion and does not clamp any data. In some cases this blend mode is less prone to artifacts than “color”.
+
+## RGB channels
+
+The following are available when blending in RGB color spaces only.
+
+RGB red channel
+: Mix the "red" channel from the input and output images, while taking the other channels unaltered from the input image.
+
+RGB green channel
+: Mix the "green" channel from the input and output images, while taking the other channels unaltered from the input image.
+
+RGB blue channel
+: Mix the "blue" channel from the input and output images, while taking the other channels unaltered from the input image.
+
+# HSV channels
+
+The following are available when blending in RGB color spaces only.
 
 HSV lightness
```

Comment:
These are HSL, not HSV (V = max(RGB), L = (min(RGB) + max(RGB)) / 2).

Reply:
ok but they're all **called** HSV so what do you want me to say here?

Reply:
They should all be called HSL.

Reply:
Forget about that, sorry for the noise.